pynets/stats/embeddings.py

```diff
@@ -452,12 +478,20 @@ def build_masetome(est_path_iterlist, ID):
                 pop_list.append(mat)
         if len(pop_list) != len(pairs):
             continue
-        atlas = prune_suffices(pairs[0].split("/")[-3])
-        res = prune_suffices("_".join(pairs[0].split(
-            "/")[-1].split("modality")[1].split("_")[1:]).split("_est")[0])
-        if "rsn" in res:
-            subgraph = res.split("rsn-")[1].split('_')[0]
-        else:
+        try: #For standard file formatting
+            atlas = prune_suffices(pairs[0].split("/")[-3])
+            res = prune_suffices("_".join(pairs[0].split(
+                "/")[-1].split("modality")[1].split("_")[1:]).split("_est")[0])
+            if "rsn" in res:
+                subgraph = res.split("rsn-")[1].split('_')[0]
+            else:
+                subgraph = "all_nodes"
+        except: #For nonstandard file formatting
+            print(
+                "WARNING: Non-standard file naming detected. "
+                "Using default values for embedding."
+                )
+            atlas = "Default"
```

Comment:
So "Default" actually refers to a specific brain network -- i.e. the "Default Mode Network". Maybe we could have the except clause fall-back to atlas = 'Unknown' instead?

Reply:
Ok, great! I didn't realize that- would you also want to change the atlas names used in the unit tests? Right now they're also set to Default, although if I understand correctly they don't actually have any impact on the test results